R/connect_email.R

```diff
@@ -64,7 +64,7 @@ attach_connect_email <- function(email = NULL,
 
       if (isTRUE(preview)) {
 
-        html_file <- tempfile(fileext = ".html")
+        html_file <- ".rsc_email.html"
```

Comment:
`.rsc_email.html` is the whole filename? Is there a way for us to use the same base name as the currently-being-knit document?

Reply:
I was just testing this as a first pass. Works well. Next step is to get the base name of the rendered document.

Reply:
I don't think we have access to the report name from within the `attach_connect_email()` function. We get only the rendered email object as input. Maybe an advantage to reusing the same name is less proliferation of these preview .html files?